docs/v0.3.X/docs/migration.md

```diff
@@ -0,0 +1,50 @@
+# Migration from v0.X to v1.X
+
+To upgrade {{v0X.crowdsec.name}} from v0.X to v1, we'll follow those steps
+
+#### Backup up configuration
+
+```
+sudo cscli backup save /tmp/crowdsec_backup
+sudo cp -R  /etc/crowdsec/config/patterns /tmp/crowdsec_backup
+```
+
+#### Uninstall old version & install new 
+
+Download latest V1 {{v0X.crowdsec.name}} version [here]({{v0X.crowdsec.download_url}})
+
+```
+tar xvzf crowdsec-release.tgz
+cd crowdsec-v1.X/
+sudo ./wizard.sh --uninstall
+sudo ./wizard.sh --bininstall
+```
+
+!!! warning
+        Don't forget to remove {{v0X.metabase.name}} dashboard if you installed it manually (without {{v0X.cli.name}}).
+
+#### Restore configuration
+
+Copy `machine_id` and `password` from `/tmp/crowdsec_backup/api_creds.json` and edit `/etc/crowdsec/online_api_credentials.yaml`
+
+```
+url: https://api.crowdsec.net/
+login: <machine_id>
+password: <password>
+```
+
+!!! warning
+        Before restoring old backup, if you have `local` or `tainted` postoverflows, be aware that they are no longer compatible. You should update the syntax (the community and us are available to help you doing this part).
+```
+sudo cscli hub update
+sudo cscli config restore --old-backup /tmp/crowdsec_backup/
+sudo cp -R /tmp/crowdsec_backup/patterns /etc/crowdsec/
+```
```

Comment:
I think it's worth to add a checklist to be sure the user didn't broke stuff, ie. `cscli capi status` and `cscli lapi status`